week3_HM/todo/src/components/TodoList.js

```diff
@@ -0,0 +1,79 @@
+import React, { Component } from 'react'
+import input from './buttons'
+import InsertTasks from './insertTasks'
+//import DatePicker from 'react-datepicker';
+//import moment from 'moment';
+
+const data = require('../todo.json')
+
+export default class TodoList extends Component{
+    state = {
+     mineTasks: data
+    }
+    handleChecked = (index) => {
```

Comment:
Index is not a very safe way to be looking up items in an array. Please refactor to use ID.